src/schema/ecommerce/create_order_mutation.js

```diff
@@ -87,6 +87,8 @@ export const CreateOrderMutation = mutationWithClientMutationId({
             createdAt
             updatedAt
             createdAt
```

Comment:
Not related to this PR but should `createdAt` be in here twice?

Reply:
Probably a merge thing. Removed it in another PR. Thanks for catching it.

Reply:
Thanks @sepans!